platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/ui/admin/api/impl/SystemPropertiesAdmin.java

```diff
@@ -234,7 +257,20 @@ public void writeSystemProperties(Map<String, String> updatedSystemProperties) {
                 LOGGER.warn("Unable to write system user attribute file for hostname update.", e);
             }
         }
+    }
 
+    /**
+     * Overwrite attributes with those from GuestClaimsHandlerExt so system high is set.
+     */
+    private void addGuestClaimsProfileAttributes(Map<String, Object> json) {
+        Map<String, Object> selectedProfileAttributes =
+                guestClaimsHandlerExt.getSelectedClaimsProfileAttributes();
```

Comment:
System high should be able to be different from the guest markings. The idea is that there is a range between these two values in some cases that would allow filtering to occur. Setting guest == system high in all cases effectively turns off filtering for all cases.